### PR TITLE
Add November blog post

### PR DESCRIPTION
**Pull request by saragon02**
:heart: Thank you for your contribution!

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [x] I've added relevant test cases.
- [x] I've added relevant documentation.
- [x] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/howtos/i18n/).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).
    * If you have added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Frontend**

- [x] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines.
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines.
- [x] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines.

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

**Comment by mfenner** [comment]
LGTM. I don't think I need to be an author on this one .

**Comment by saragon02** [comment]
Thanks @mfenner. I can remove your name if you prefer; otherwise I'm happy to keep it to acknowledge your managing editor status on the blog.

**Comment by mfenner** [comment]
Ok. Then keep my name. In my InvenioRDM instance I am working on other roles for blog post contributors, with `editor` the first one to add. We can aim for something similar for the Invenio blog but it will take a bit of time (and probably after implementing DOI registration).

File: content/blog/2024-11-19-november-GREI/contents.lr [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Zenodo team members are co-leading the effort to share the [Make Data Count](https://makedatacount.org/learn-about-us/) usage metrics with the wider community via DataCite 
+
+### Use Cases
+
+- Zenodo team members co-led the task group responsible for updating the [Use Case catalog](https://doi.org/10.5281/zenodo.12212654) with two new use cases: one outlining a scenario where a researcher utilizes a generalist repository in lieu of a local institutional repository, and one scenario where a researcher must deposit portions of a heterogenous dataset into several different repositories, one of which is a generalist repository
+
+### Connecting Digital Objects
+
+- Zenodo team members are working on the effort to map DataCite relationTypes to all types of non-data research objects, and to submit any needed updates to DataCite via API
+- InvenioRDM has also completed work on the FAIR signposting recommendation, thanks to Guillaume Viger’s work on [signposting](https://doi.org/10.5281/zenodo.12554416)
```
Comment (ntarocco) [inline content/blog/2024-11-19-november-GREI/contents.lr:46]: Does it make sense to add a link to `Guillaume Viger’s` words to some public bio page (or ORCID?), if such bio page exist?

**Review by ntarocco — APPROVED** [review]
LGTM! Nice work! 👏🏼

**Pull request merged**